test/board/calculate-board-vulnerability.test.ts

```diff
@@ -0,0 +1,108 @@
+import { Board, Types } from '../../src';
+import { Vulnerability } from '../../src/types';
+
+describe('Testing Board.calculateBoardVulnerability', () => {
+	it('Love all boards', () => {
+		const boardNumbers = [1, 8, 11, 14];
+		for (const board of boardNumbers) {
+			expect(Board.calculateBoardVulnerability(board)).toStrictEqual(
+				Vulnerability.NvNv
+			);
+		}
+	});
+	it('NS Vul boards', () => {
+		const boardNumbers = [2, 5, 12, 15];
+		for (const board of boardNumbers) {
+			expect(Board.calculateBoardVulnerability(board)).toStrictEqual(
+				Vulnerability.VNv
+			);
+		}
+	});
+	it('EW Vul boards', () => {
+		const boardNumbers = [3, 6, 9, 16];
+		for (const board of boardNumbers) {
+			expect(Board.calculateBoardVulnerability(board)).toStrictEqual(
+				Vulnerability.NvV
+			);
+		}
+	});
+	it('Game all boards', () => {
+		const boardNumbers = [4, 7, 10, 13];
+		for (const board of boardNumbers) {
+			expect(Board.calculateBoardVulnerability(board)).toStrictEqual(
+				Vulnerability.VV
+			);
+		}
+	});
```

Comment:
Can we add a case for passing in a negative board number, which will then cause it to throw an Error.